website/docs/r/batch_pool.html.markdown

```diff
@@ -120,6 +120,21 @@ The following arguments are supported:
 
 * `storage_image_reference` - (Required) A `storage_image_reference` for the virtual machines that will compose the Batch pool.
 
+~> **NOTE:** It's possible to reference a custom VM image for the pool by specifying it's `id` in the `id` property of the `storage_image_reference`. This property is mutually exclusive with other properties. The VM image should be in the same region that the batch pool you are about to create. See [official documentation](https://docs.microsoft.com/en-us/azure/batch/batch-custom-images) for more details.
+
+```
```

Comment:
Shouldn't this code sample be moved to a new section under the **Example Usage** section? Say something like ## Example Usage with Custom Image?

Reply:
Yes, totally agree.